src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/dto/SavedMetadataErrorResponse.java

```diff
@@ -0,0 +1,19 @@
+package ca.corefacility.bioinformatics.irida.ria.web.projects.dto;
+
+import ca.corefacility.bioinformatics.irida.ria.utilities.SampleMetadataStorage;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.ajax.AjaxResponse;
+
+/**
+ * Returns the SampleMetadataStorage on error.
```

Comment:
Isn't this the same as `SavedMetadataException`?  They seem to do the exact same thing so just have one.

Reply:
We will keep them as they have slightly different useage.